contracts/auction-handler/FastLaneAuctionHandler.sol

```diff
@@ -519,18 +416,29 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
     |             Modifiers             |
     |__________________________________*/
 
-    modifier whenNotPaused() {
-        if (paused) revert RelayPermissionPaused();
+    modifier onlyCurrentValidator() {
+        require(msg.sender != address(0), "msg.sender = 0 address");
```

Comment:
```suggestion
```
See comment below/above, effectively impossible, no need to check

Reply:
Ben comment is true, msg.sender will never be address(0)